SPECS/keras/keras.spec

```
@@ -69,6 +71,9 @@ bazel --batch build  --verbose_explanations //keras/tools/pip_package:build_pip_
 
 
 %changelog
+* Tue Aug 01 2023 Riken Maharjan <rmaharjan@microsoft.com> - 2.11.0-2
+- remove .bazelversion file.
```

Nit-pick for here and other spec files:
```suggestion
- Remove .bazelversion file.
```